keyboards/xiudi/xd75/keymaps/raoeus/keymap.c

```diff
@@ -0,0 +1,150 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "keymap_steno.h"
+
+void matrix_init_user(void) {
+    steno_set_mode(STENO_MODE_GEMINI);
+}
+
+enum layer_names { _QWERTY, _LOWER, _RAISE, _BOTH, _STENO };
+
+enum keycodes { QWERTY = SAFE_RANGE, LOWER, RAISE, STENO, EXT_STN };
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* QWERTY
+     * .--------------------------------------------------------------------------------------------------------------------------------------.
+     * | `      | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | Bksp   | NUM    | /      | *      |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+     * | TAB    | Q      | W      | E      | R      | T      | Y      | U      | I      | O      | P      | Del    | 7      | 8      | 9      |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+     * | Esc    | A      | S      | D      | F      | G      | H      | J      | K      | L      | ;      | "      | 4      | 5      | 6      |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+     * | LShift | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Enter  | 1      | 2      | 3      |
+     * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+     * | LCtl   | LAlt   | LGUI   | Steno  | Lower  | Space  | Space  | Raise  | Left   | Down   | Up     | Right  | 0      | .      | Enter  |
+     * '--------------------------------------------------------------------------------------------------------------------------------------'
+     */
+    [_QWERTY] = LAYOUT_ortho_5x15(KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_BSPC, KC_NUM, KC_PSLS, KC_PAST, KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_DEL, KC_P7, KC_P8, KC_P9, KC_ESC, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_P4, KC_P5, KC_P6, KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_ENT, KC_P1, KC_P2, KC_P3, KC_LCTRL, KC_LALT, KC_LGUI, STENO, LOWER, KC_SPC, KC_SPC, RAISE, KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, KC_P0, KC_PDOT, KC_PENT),
```

Comment:
Might be worth formatting this so it's not all on one line.

Reply:
Sure thing, that's no problem. I've reformatted so it's more human readable. I also tested building/flashing again to be sure all is good.
https://github.com/qmk/qmk_firmware/pull/17456/commits/a34c2d72390fcb5651e62a308593f2883778a889